ledger-service/utils/src/main/scala/com/digitalasset/ledger/service/Grpc.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

The 2.5 branch is not up-to-date in that regard.

Reply:
822369dd89a